### PR TITLE
#2 setup triggers, set target and cleanup code

### PR DESCRIPTION
**Pull request by dheerajdlalwani**

### CODE REVIEW

File: postpone/aws_cli_setup.py [hunk 1]
```diff
@@ -6,6 +6,24 @@
...
+#         function_details = json.loads(result.stdout)
+
+#         # Extract the Function ARN
+#         function_arn = function_details['Configuration']['FunctionArn']
+
+#         return function_arn
+#     except subprocess.CalledProcessError as e:
+#         print(f"Error executing AWS CLI command: {e}")
+#         return None
+
```
Comment (dheerajdlalwani) [inline postpone/aws_cli_setup.py:26]: remove commented code.

File: lambda_function.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+def handler(request):
```
Comment (dheerajdlalwani) [inline lambda_function.py:1]: pass the 2nd parameter like chintan tested on AWS

File: postpone/aws_cli_setup.py [hunk 3]
```diff
@@ -71,13 +91,75 @@ def configure_aws():
...
+    rule_name=str(uuid.uuid4()).replace('-','')
+    subprocess.run(f'aws events put-rule --schedule-expression "rate({minutes} {rate_unit})" --name {rule_name}',text=True, capture_output=True,shell=True)
+    time.sleep(3)
+    set_lambda_rule(function_arn,rule_name)
+    time.sleep(3)
+    result=setup_trigger(function_name,rule_name)
+    # select target
+    # Add other tasks as needed (e.g., update_lambda_function, list_lambda_functions, create_new_rule, etc.)
+
+# ... (other functions)
```
Comment (dheerajdlalwani) [inline postpone/aws_cli_setup.py:162]: clean-up comments

File: postpone/setup.py [hunk 4]
```diff
@@ -2,7 +2,7 @@
 print(find_packages())
 setup(
     name='aws-cli-setup',
-    version='2.5',
+    version='5.0',
```
Comment (dheerajdlalwani) [inline postpone/setup.py:5]: make version as 1.0

**Review by dheerajdlalwani — COMMENTED** [review]
Make these changes @whovivekshukla then we can merge.

**Review by chintanpuggalok — APPROVED** [review]

**Pull request merged**